src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/DeleteResultSet.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.executor.format;
+
+import com.amazon.opendistroforelasticsearch.sql.domain.Delete;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.index.reindex.BulkByScrollResponse;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class DeleteResultSet extends ResultSet {
```

Comment:
Probably not necessary for this data class. But it would be better to add a UT for test coverage.

Reply:
Sure, will add unit tests.

Reply:
Thanks!

Reply:
Added unit tests.